### PR TITLE
Register jiyath.is-a.dev

### PR DESCRIPTION
**Pull request by Jiyath5516F**
Register jiyath.is-a.dev with A record pointing to 185.199.108.153.

### CODE REVIEW

**Review by MaskDuck — CHANGES_REQUESTED** [review]

File: domains/jiyath.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Jiyath5516F",
+           "email": "",
+           "discord": "831836517516312586",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.Ou0aZJPtQR6dKH2nzSJA1hCmd52TrzxzUZWRL1JTg9EYUmyYBhVklrkaDqGV_aV8gKg64q7RbQ8Aa9Ghd5zI72IMp_k8noQQzr3i5DNv1ij001AtGFuUS7C_7nELc4hBrmJdVt02d8iH2CAM3Qg6jW1dNrSLRF6SMlU6Nx91ruWIEUayzHz30kG6mS9l-0SxJwMpDvmO8tsFqumPy_wqDQXePnqNwVGQK3UeZscSXYTXEQrVaJKgrjKzwJovgnrussMRO6Mg6bVQcMp80JFts9mXtr5DQ7ZBG2o-WY4y2X7ee1aFmAvJgtw8z24asocNYiYVKy4d7rzuTS9AIIJe-Q.qoTJuB_rN7r61yHm_SQ1sQ.pwH7RHNzGpV365BBO_tolwajA2XArfIV0gdLJR0y40VRgabi70frRH49rx62T_n3voJu11dQeX4-PusUA21JK3ATebpwo557CA9Za9LY3hw.JGAHTASzBs0rdG7I7qLs7Q"
+        },
+    
+        "record": {
+            "A": ["185.199.108.153"]
```
Comment (Stef-00012) [inline domains/jiyath.json:10]: I can not reach your website, 404, if you use github pages, you need to put all 4 IPs

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/jiyath.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Jiyath5516F) [reply domains/jiyath.json:10]: https://jiyath5516f.github.io/jiyath.is-a.dev/
any possibilities to add 4 A records with this link?

Comment (Jiyath5516F) [reply domains/jiyath.json:10]: https://jiyath5516f.github.io/jiyath.is-a.dev/

[the remark this answers is not in the document]
Comment (DEV-DIBSTER) [reply domains/jiyath.json:10]: You would need to add them all to the array:

```json
{
      "A": ["185.199.108.153", "INSERT_ADDITONAL_IP", "INSERT_ADDITONAL_IP", "INSERT_ADDITONAL_IP"]
}
```

[the remark this answers is not in the document]
Comment (Jiyath5516F) [reply domains/jiyath.json:10]: Actually, I don't have 4 IP addresses, so I will close this PR and create one with a CNAME.

**Pull request closed without merging**